src/data/helpText.yaml

```diff
@@ -1,29 +1,28 @@
 product:
-  name: ""
+  name: "Name of offering (ie. your product or service)."
```

Comment:
Nitpick here, should be either `ie` no period or `i.e.`

Reply:
Ok. Such a silly word.